src/controllers/messagesController.ts

```diff
@@ -29,55 +21,29 @@ export default class MessagesController {
    * @param req
    * @param res
    */
-  public getUserMessages(req: express.Request, res: express.Response): void {
+  public getMessagesByUser(req: express.Request, res: express.Response): void {
     const maybeUser = extractUserFromRequest(req);
 
-    maybeUser.fold(
-      () => {
+    if (isLeft(maybeUser)) {
```

Comment:
I suggest to call it errorOrUser when it's an Either<Error, User> and maybeUser when it's an Option<User>
(as sometimes you'll get an Either<Error, Option<User>> which will be errorOrMaybeUser : )